test/adapters/prebid.spec.js

```diff
@@ -991,6 +999,26 @@ describe('ADAPTER: Prebid', function() {
             delete CONF.adapters[adapterID][CONSTANTS.CONFIG.SERVER_SIDE_ENABLED];
             done();
         });
+
+        xit('should set config for single request partners present in constants',function(done){
+            var expectedResult = {
+                "rubicon":{
+                    singleRequest:true
+                }
+            }
+            PREBID.fetchBids(activeSlots, impressionID);
+            window["owpbjs"].setConfig.should.be.called.with(expectedResult);
+            var prebidConfig = window["owpbjs"].getConfig();
+            expectedResult.should.be.equal(prebidConfig);
+            done();
+        });
+
+        xit('should not have set config for single request parnter if it is not present for bidding',function(done){
+            partnerToEnableSRA = [];
```

Comment:
partnerToEnableSRA variable is local to the case it may not be impacting the code flow in the actual code that we want to test.

Reply:
Agreed.
Removed.